topic_name_lidar: "/points_raw"
box_filter_set:
  center: [9, 2, 0.5]
  cube_side_length: [1,1,10]
intensity_filter_set:
  lower_upper_bound: [20, 150]
target_size_set:
  model_type: triangle
  center_to_end_length: 0.6
  depth: 0.163299
edge_points_resolution: 20

